contributing.md

```diff
@@ -289,6 +289,49 @@ ansible-playbook --skip-tags tests::cleanup \
     -i testhost, tests/playbooks/tests_802_1x.yml
 ```
 
+### Integration tests with podman
+1. Create ``~/.ansible/collections/ansible_collections/containers/podman/`` if this
+directory does not exist and ``cd`` into this directory.
+```
```

Comment:
```suggestion
```bash
```

Would be nice to mention the interpreter for the code snippets for better syntax highlighting

Reply:
I see the effect when I follow your suggestion, you are absolutely right

Reply:
updated